src/web/styles.scss

```diff
@@ -49,6 +49,19 @@ body {
   padding: 30px 20px;
 }
 
+.table {
+  margin-bottom: 0;
+}
+
+.table th,
+.table td {
+  padding: 0.5rem
```

Comment:
I think group selector can be put on the same line.

Reply:
Right, but seems like our ts-ling config requires new lines after the comma